packages/graphql-testing/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 6.4.1
+
+### Patch Changes
+
+- [#2543](https://github.com/Shopify/quilt/pull/2543) [`7c8b83134`](https://github.com/Shopify/quilt/commit/7c8b831346ec84150dd8ed178741b39408fb0fe8) Thanks [@acmertz](https://github.com/acmertz)! - Update resolveAll() to accept a custom filter option to selectively resolve requests.
```

Comment:
This should have been a minor version bump since it's introducing a new feature.

Reply:
This PR to do the release has not yet been merged. Go open up a PR that changes `patch` to `minor` in [the changeset that maps to this update](https://github.com/Shopify/quilt/blob/main/.changeset/twenty-needles-juggle.md?plain=1#L2).